source/index.md

```diff
@@ -50,24 +50,47 @@ Explanations of history and ideas in the Nix ecosystem
 :::
 ::::
 
-## What can you do with Nix?
-
-The following list illustrates some of what can be achieved with the Nix ecosystem:
-
-- {ref}`Reproducible development environments <ad-hoc-envs>`.
-- Easy installation of software over URLs.
-- Easy transfer of software environments between computers.
-- {ref}`Declarative specification of Linux machines <deploy-nixos-using-terraform>`.
-- {ref}`Reproducible integration testing using virtual machines <integration-testing-vms>`.
-- Avoidance of version conflicts with already installed software.
-- Installing software from source code.
-- {ref}`Transparent build caching using binary caches <github-actions>`.
-- Strong support for software auditability.
-- {ref}`First-class cross compilation support <cross-compilation>`.
-- Remote builds.
-- Remote deployments.
-- Atomic upgrades and rollbacks.
-
+## What is Nix?
+
+[Nix](https://github.com/NixOS/nix) allows treating Unix processes and file system operations as pure functions.
+
+It is the basis of an ecosystem of exceptionally powerful tools.
+[Nixpkgs](https://github.com/nixos/nixpkgs) is [the largest, most up-to-date software repository in the world](https://repology.org/repositories/graphs).
+[NixOS](https://github.com/NixOS/nixpkgs/tree/master/nixos) is a Linux distribution that can be configured fully declaratively, with unmatched flexibility.
+
+## How to use software from Nixpkgs
+
+This is what you can do with software from Nixpkgs:
+- Run **standalone programs** locally with Nix
+- Use **libraries or tools** to build software with Nixpkgs
+- Deploy **services** to machines running NixOS
+
+In order to do that:
+- [Install Nix on Linux or WSL](https://nix.dev/install-nix)
+
+It will help you to go more quickly if you learn to:
+- [Read the Nix language](https://nix.dev/tutorials/nix-language)
+- [Package existing software with Nixpkgs](https://nix.dev/tutorials/packaging-existing-software)
+- [Work with NixOS modules](https://nix.dev/tutorials/module-system/)
+- [Run NixOS in virtual machines](https://nix.dev/tutorials/nixos/nixos-configuration-on-vm)
+- [Provision remote NixOS machines via SSH](https://nix.dev/tutorials/nixos/provisioning-remote-machines)
+- [Set up your own cache for sharing binaries](https://nix.dev/tutorials/nixos/binary-cache-setup)
+
```

Comment:
```suggestion
```

Let the big link boxes direct readers to the next steps. Doesn't make sense to hard-code navigation here, that's what actual navigation is for.